src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -339,60 +339,4 @@ public void testIsTypeDeclaration() {
                 .that(TokenUtil.isTypeDeclaration(TokenTypes.RECORD_DEF))
                 .isTrue();
     }
-
-    @Test
-    public void testIsOfTypeTrue() {
-        final int type = TokenTypes.LITERAL_CATCH;
-        final DetailAstImpl astForTest = new DetailAstImpl();
-        astForTest.setType(type);
-        final boolean result1 = TokenUtil.isOfType(type, TokenTypes.LITERAL_FOR,
-                                TokenTypes.LITERAL_IF, TokenTypes.LITERAL_CATCH);
-        final boolean result2 = TokenUtil.isOfType(astForTest, TokenTypes.LITERAL_FOR,
-                                TokenTypes.LITERAL_IF, TokenTypes.LITERAL_CATCH);
-
-        assertWithMessage("Token type did not match")
-                .that(result1)
-                .isTrue();
-        assertWithMessage("Token type did not match")
-                .that(result2)
-                .isTrue();
-    }
-
-    @Test
-    public void testIsOfTypeFalse() {
-        final int type = TokenTypes.LITERAL_CATCH;
-        final DetailAstImpl astForTest1 = new DetailAstImpl();
-        final DetailAstImpl astForTest2 = null;
-        astForTest1.setType(type);
-        final boolean result1 = TokenUtil.isOfType(type, TokenTypes.LITERAL_FOR,
-                                TokenTypes.LITERAL_IF, TokenTypes.LITERAL_ELSE);
-        final boolean result2 = TokenUtil.isOfType(astForTest1, TokenTypes.LITERAL_FOR,
-                                TokenTypes.LITERAL_IF, TokenTypes.LITERAL_ELSE);
-        final boolean result3 = TokenUtil.isOfType(astForTest2, TokenTypes.LITERAL_FOR,
-                                TokenTypes.LITERAL_IF, TokenTypes.LITERAL_ELSE);
-
-        assertWithMessage("Token type should not match")
-                .that(result1)
-                .isFalse();
-        assertWithMessage("Token type should not match")
-                .that(result2)
-                .isFalse();
-        assertWithMessage("Token type should not match")
-                .that(result3)
-                .isFalse();
-    }
-
-    @Test
-    public void testIsBooleanLiteralType() {
```

Comment:
this test is not related to isOfType(), you should not remove it

Reply:
Changes Done

Reply:
Not done, I still see removal.
It might be reason of CI failures.

It is ok to remove only if does not affect CI

Reply:
I have already added the code again. Please checkout.